### ISSUE TITLE
Rectangle keys don't work after mac sleep

### ISSUE THREAD

**Issue by andrew-kay-oxb** [rxhanson/Rectangle#1244]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version:  Ventura 13.4.1
Rectangle version: 0.70 (76)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 
lots of these:
```
2023-08-18T10:40:51+01:00: Unable to obtain the accessibility element with the specified attribute at mouse location
2023-08-18T10:40:52+01:00: Unable to obtain window id
```
Rectangle used to work perfectly with Monterey (thank you -- brilliant tool.) I updated recently to Ventura, and problems began.  After mac has been asleep for a significant period and needs to be woken up with password/login again, Rectangle doesn't operate.  Keys don't work, and menu options do nothing.  Quit and restart rectangle doesn't help.  _Short_ sleeps (woken up with touch id) are ok, as is reboot.
Thanks

**Comment by rxhanson** [comment]
Thanks for reporting. This means that macOS is not providing the necessary accessibility information to Rectangle. Unfortunately, this one appears to be a macOS issue. Here's a similar issue: https://github.com/rxhanson/Rectangle/issues/640. There's a workaround in that issue that might work for you? Not great, but better than a reboot every time.
